lib/bolt/core/src/tasks/config/generate.rs

```diff
@@ -309,6 +309,16 @@ pub async fn generate(project_path: &Path, ns_id: &str) -> Result<()> {
 			Ok(value(generate_password(32)))
 		})
 		.await?;
+	generator
+		.generate_secret(&["crdb", "readonly", "username"], || async {
+			Ok(value("rivet"))
```

Comment:
nit: needs naming based on who's reading it.

so: crdb/grafana/username -> grafana